server/routes/settings.js

```diff
@@ -0,0 +1,134 @@
+const express = require("express");
+const router = express.Router();
+const mongoose = require("mongoose");
+const SettingsModel = require("./../models/Settings");
+const UserModel = require("./../models/User");
+const { jwtVerify } = require("../utils/authUtils");
+
+mongoose.connect(process.env.MONGO_URL, { useNewUrlParser: true  });
+
+const saveSettings = (settings, res) => {
+  settings.save((err) => {
+    if (err) {
+      console.log(err);
+    }
+  });
+  res.send({ error: false, settings });
+}
+
+router.post("/settings/:email", jwtVerify, (req, res, next) => {
+  UserModel.findOne({ username: req.params.email },
+    (err, user) => {
+      if (err) {
+        console.log(err);
```

Comment:
For all the errors, you can feed them into the next() function as I've set up error handling middleware. The error handler just takes a string so you can just call next(err) here.